Update xamarin.md (#2689)
dotnet/EntityFramework.Docs: entity-framework/core/get-started/xamarin.md

}
```

## iPhone Configuration

> Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:

Reviewer: interpretor -> interpreter

> Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:

Reviewer 2: ```suggestion
Entity Framework Core sometimes generates dynamic code for performance reasons. This isn't supported on the iOS platform and may cause the app to crash. The Xamarin team built an interpreter to handle these scenarios. It should be configured in the `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
```

Reviewer 2: Per MS style guide, prefer contractions. Also the style guide insists files be `*italic*`, but for project files, I prefer code fencing as you've done.

```xml
<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhone' ">
</PropertyGroup>
```

Then add the `--interpreter` flag as an extra argument inside the group:

> ```xml

Reviewer 3: Combining both xml blocks would make it clearer what is happening overall. Also easy to copy paste.

> <CodesignExtraArgs>--interpreter</CodesignExtraArgs>

Reviewer: Shouldn't it be `<MtouchExtraArgs>--interpreter</MtouchExtraArgs>`?
See my comment in #2666

Author: Good question. I'm confirming with @codemillmatt who has been working on the sample.

Reviewer 4: Turns out the interpreter isn't the issue here after all - it is a linking issue. I'll update the code to make sure EF isn't linked out.

```

> More information is available in the [interpreter repository](https://github.com/spouliot/interpreter).

Reviewer 3: Is it microsoft product? If not we should call out accordingly.

Author: Good question. It was built by someone on the Xamarin team but it is in their personal repo. Do we have a standard boilerplate?
